src/EnergyPlus/HeatPumpWaterToWaterSimple.cc

```diff
@@ -780,98 +693,116 @@ namespace HeatPumpWaterToWaterSimple {
-                rho = GetDensityGlycol(PlantLoop(GSHP(GSHPNum).LoadLoopNum).FluidName,
-                                       DataGlobals::CWInitConvTemp,
-                                       PlantLoop(GSHP(GSHPNum).LoadLoopNum).FluidIndex,
-                                       RoutineName);
-                GSHP(GSHPNum).LoadSideDesignMassFlow = GSHP(GSHPNum).RatedLoadVolFlowHeat * rho;
-                rho = GetDensityGlycol(PlantLoop(GSHP(GSHPNum).SourceLoopNum).FluidName,
-                                       DataGlobals::CWInitConvTemp,
-                                       PlantLoop(GSHP(GSHPNum).SourceLoopNum).FluidIndex,
-                                       RoutineName);
-                GSHP(GSHPNum).SourceSideDesignMassFlow = GSHP(GSHPNum).RatedSourceVolFlowHeat * rho;
-            } else if (GSHP(GSHPNum).WWHPPlantTypeOfNum == TypeOf_HPWaterEFCooling) {
-                rho = GetDensityGlycol(PlantLoop(GSHP(GSHPNum).LoadLoopNum).FluidName,
-                                       DataGlobals::CWInitConvTemp,
-                                       PlantLoop(GSHP(GSHPNum).LoadLoopNum).FluidIndex,
-                                       RoutineName);
-                GSHP(GSHPNum).LoadSideDesignMassFlow = GSHP(GSHPNum).RatedLoadVolFlowCool * rho;
-                rho = GetDensityGlycol(PlantLoop(GSHP(GSHPNum).SourceLoopNum).FluidName,
-                                       DataGlobals::CWInitConvTemp,
-                                       PlantLoop(GSHP(GSHPNum).SourceLoopNum).FluidIndex,
-                                       RoutineName);
-                GSHP(GSHPNum).SourceSideDesignMassFlow = GSHP(GSHPNum).RatedSourceVolFlowCool * rho;
+            this->reportPower = 0.0;
+            this->reportEnergy = 0.0;
+            this->reportQLoad = 0.0;
+            this->reportQLoadEnergy = 0.0;
+            this->reportQSource = 0.0;
+            this->reportQSourceEnergy = 0.0;
+            this->reportLoadSideMassFlowRate = 0.0;
+            this->reportLoadSideInletTemp = 0.0;
+            this->reportLoadSideOutletTemp = 0.0;
+            this->reportSourceSideMassFlowRate = 0.0;
+            this->reportSourceSideInletTemp = 0.0;
+            this->reportSourceSideOutletTemp = 0.0;
+            this->IsOn = false;
+            this->MustRun = true;
+
+            if (this->WWHPPlantTypeOfNum == TypeOf_HPWaterEFHeating) {
+                rho = GetDensityGlycol(
+                    PlantLoop(this->LoadLoopNum).FluidName, DataGlobals::CWInitConvTemp, PlantLoop(this->LoadLoopNum).FluidIndex, RoutineName);
+                this->LoadSideDesignMassFlow = this->RatedLoadVolFlowHeat * rho;
+                rho = GetDensityGlycol(
+                    PlantLoop(this->SourceLoopNum).FluidName, DataGlobals::CWInitConvTemp, PlantLoop(this->SourceLoopNum).FluidIndex, RoutineName);
+                this->SourceSideDesignMassFlow = this->RatedSourceVolFlowHeat * rho;
```

Comment:
@Myoldmopar should you change these to HWInitConvTemp now?

Reply:
I can, but it may cause diffs in a few files.  My intention was to make a small branch to address @mjwitte's comments without being invasive or difficult to review.  I'll go ahead and make that change but note that diffs will likely occur similar to the ones that occurred on this original branch.